packer/scylla_install_image

```diff
@@ -112,6 +113,16 @@ if __name__ == '__main__':
         run('dpkg -i /tmp/amazon-ssm-agent.deb', shell=True, check=True)
         run('systemctl enable amazon-ssm-agent', shell=True, check=True)
 
+        run('apt-get install -y linux-aws-lts-22.04', shell=True, check=True)
```

Comment:
if you are going to backport to older release branches.... you should be careful not to forgot to match it to the correct version....

Reply:
> I'm approving this now, but I think we should replace shell scripting techniques (like awk, grep, head...) to python code, I will send a patch for it later.

Modified

Reply:
let's start with 5.2 and 2023.1